### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by sanjaymhj**
Notifications for each vaccination while adding a new child.
Deleting the notification if vaccination is taken.
Default image for boy and girl and a default cover image.
P.S Notification time is in the date of vaccination in 8 am
@laxmankhanal @anmol-lf @bishalg

### CODE REVIEW

File: Vaccination/ViewController/MainVC.swift [hunk 1]
```diff
@@ -52,38 +75,69 @@ class ViewController: UIViewController {
...
+    override func touchesBegan(touches: Set<UITouch>, withEvent event: UIEvent?) {
+        self.view.resignFirstResponder()
+    }
+
+    func updateChildToShow() {
+        self.childNameLabel.text = currentChild!.firstName!+" "+currentChild!.lastName!
+        self.childAgeLabel.text = String(Int(calculateAge(withDOB: currentChild!.dateOfBirth!)))+" days"
+        self.tableData = segmentSelected.getTableData(currentChild!)
+        self.vaccinationListTVC.reloadData()
+        self.profilePhoto.image = UIImage(data: (currentChild?.photo)!)
```
Comment (bishalg) [inline Vaccination/ViewController/MainVC.swift]: these codes has lots of bangs ( !  ) minimize then by checking the value at first
add space after and before the  +  operator

File: Vaccination/Constants/ImageConstants.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        
+    }
+    
+    static var girlImage: UIImage {
+        get {
+            return UIImage(named: "girl")!
+        }
+        
+    }
+}
```
Comment (bishalg) [inline Vaccination/Constants/ImageConstants.swift]: new line at the end of file

Comment (sanjaymhj) [inline Vaccination/Constants/ImageConstants.swift]: :ok:

File: Vaccination/ViewController/MainVC.swift (hunk 1 above)
Comment (sanjaymhj) [inline Vaccination/ViewController/MainVC.swift]: :ok:

**Pull request merged**